File: src/codes/Users_Manual/silencer/silencer_fixed_update_rate.cpp
//~#include<autd3.hpp>
//~int main() {
const auto update_rate_intensity = 256;
const auto update_rate_phase = 256;
const auto config =
    autd3::Silencer::from_update_rate(update_rate_intensity, update_rate_phase);
//~return 0; }

File: src/codes/Users_Manual/silencer/silencer_fixed_update_rate.rs
# use autd3::prelude::*;
# #[allow(unused_variables)]
# fn main() -> Result<(), Box<dyn std::error::Error>> {
let update_rate_intensity = 256;
let update_rate_phase = 256;
let config = Silencer::from_update_rate(update_rate_intensity, update_rate_phase);
# Ok(())
# }